packages/radio-group/src/LionRadioGroup.js

```diff
@@ -36,7 +36,7 @@ export class LionRadioGroup extends LionFieldset {
 
   get serializedValue() {
     const el = this._getCheckedRadioElement();
-    return el ? el.serializedValue : '';
+    return el ? el.serializedValue : {};
```

Comment:
I think it would make sense that this is an empty string. The serialized value of a radio group should return the checked radio element's serializedValue, which should be a string, so in case there is no selected radio element, it should return empty string.

Also could you amend your last commit where you add the tests to a chore? Otherwise we get 2 bugfix entries in the changelog ;).